dist/scripts/main.js

```diff
@@ -0,0 +1,157 @@
+  'South': ['Kherson', 'Mykolaiv', 'Odesa', 'Zaporizhzhia'],
+  'West':
+    ['Chernivtsi', 'Ivano-Frankivsk', 'Khmelnytskyi', 'Lutsk', 'Lviv', 'Rivne',
+      'Ternopil', 'Uzhhorod'],
+};
+
+function main() {
+  const registrationForm = document.forms.registrationForm;
+  const fullName = document.registrationForm.fullName;
+  const needMore = document.registrationForm.needMore;
+  const phoneNumber = document.registrationForm.phoneNumber;
+  const region = document.registrationForm.region;
+  const city = document.registrationForm.city;
+  const submitButton = registrationForm.querySelector('input[type="submit"]');
+
+  const initBlankInSelect = function(selectElement) {
+    const blankOption = document.createElement('option');
+
+    selectElement.options.length = 0;
+    blankOption.value = 'blank';
+    blankOption.text = '';
+    selectElement.add(blankOption, null);
+  };
+
+  const validateAll = function() {
+    let isValid;
+    const backgroundRed = function(someInput) {
+      someInput.classList.remove('inputs-column_valid');
+      someInput.classList.add('inputs-column_invalid');
+    };
+    const backgroundGreen = function(someInput) {
+      someInput.classList.remove('inputs-column_invalid');
+      someInput.classList.add('inputs-column_valid');
+    };
+    const emptied = function(someInput) {
+      someInput.classList
+        .remove('inputs-column_invalid', 'inputs-column_invalid');
+    };
+
+    isValid = fullNameRegExp.test(fullName.value.trim());
+    if (fullName.value !== '') {
+      if (isValid) {
+        backgroundGreen(fullName);
+      } else {
+        backgroundRed(fullName);
+      }
+    } else {
+      emptied(fullName);
+    }
+    if (!needMore.checked) {
+      const nextResult = phoneNumberRegExp.test(phoneNumber.value.trim());
+      if (phoneNumber.value !== '') {
+        if (nextResult) {
+          backgroundGreen(phoneNumber);
+        } else {
+          backgroundRed(phoneNumber);
+        }
+      } else {
+        emptied(phoneNumber);
+      }
+      isValid &= nextResult;
+      isValid &= region.options.selectedIndex !== 0;
+      isValid &= city.options.selectedIndex !== 0;
+    }
+    return isValid;
+  };
+
+  const renderByCheckBox = function() {
+    phoneNumber.hidden = needMore.checked;
+    document.querySelector(`label[for='${phoneNumber.id}']`)
+      .hidden = needMore.checked;
+    region.hidden = needMore.checked;
+    document.querySelector(`label[for='${region.id}']`)
+      .hidden = needMore.checked;
+    city.hidden = needMore.checked;
+    document.querySelector(`label[for='${city.id}']`)
+      .hidden = needMore.checked;
+  };
+
+  // Initializing region and city selects
+  initBlankInSelect(region);
+  initBlankInSelect(city);
+  Object.keys(citiesByRegionsObject).forEach((someRegion) => {
+    region.options[region.options.length] = new Option(someRegion, someRegion);
+  });
+  region.addEventListener('change', (someEvent) => {
```

Comment:
someEvent sounds strange, use just `event`

Reply:
lint does not agree with this name: "'event' is already declared in the upper scope  no-shadow"